web/styles/zulip.css

```diff
@@ -2111,3 +2111,51 @@ body:not(.hide-left-sidebar) {
         border: 1px solid var(--color-border-personal-menu-avatar);
     }
 }
+
+#message-list-navigation {
+    display: flex;
+    justify-content: flex-end;
+    margin: 5px;
+    /* 18px at 16px/1em */
+    font-size: 1.125em;
+}
+
+#message-list-navigation-buttons-wrapper {
+    display: inline;
+    border: 1px solid var(--color-border-sidebar-subheader);
+    border-radius: 5px;
+}
+
+.message-list-navigation-button {
```

Comment:
A good refactor that this makes me think of is extracting a `message_feed.css` with some of the outer containing classes like `div.message-list`, `.empty_feed_notice`, `.sub-unsub-message`, `.date_row`, etc.

This CSS would belong there; would be helpful for the goal of shrinking `zulip.css`. Not a blocker for this, but maybe a fine project if you're in the mood for that style of work.